src/syft/core/node/domain/client.py

```diff
@@ -70,62 +71,60 @@ def __getitem__(self, key: Union[str, int]) -> RequestMessage:
     def __repr__(self) -> str:
         return repr(self.requests)
 
+    @property
+    def pandas(self) -> pd.DataFrame:
+        request_lines = [
+            {
+                "Requested Object's tags": request.object_tags,
+                "Reason": request.request_description,
+                "Request ID": request.id,
+                "Requested Object's ID": request.object_id,
+                "Requested Object's type": request.object_type,
+            }
+            for request in self.requests
+        ]
+        return pd.DataFrame(request_lines)
+
     def add_handler(
         self,
         action: str,
         print_local: bool = False,
         log_local: bool = False,
-        name: Optional[str] = None,
+        tags: List[str] = [],
```

Comment:
Not safe/a good practice: https://stackoverflow.com/questions/1132941/least-astonishment-and-the-mutable-default-argument

Reply:
Good point. I will change it to 
```python
tags: Optional[List[str]] = None,
```